lib/valkyrie/metadata_adapter.rb

```diff
@@ -14,11 +14,35 @@ def register(adapter, short_name)
       # Find an adapter by its short name.
       # @param short_name [Symbol]
       # @return [#persister,#query_service]
+      # @raise RuntimeError when the given short_name is not found amongst the registered adapters
       def find(short_name)
         symbolized_key = short_name.to_sym
+        # TODO: Instead of RuntimeError could we simply perform `adapters.fetch(symbolized_key)` this would raise a KeyError instead
```

Comment:
But then it wouldn't have the helpful message that the custom error provides.  Can we remove this TODO?

Reply:
Will do